unlock-app/src/hooks/usePaywallLocks.ts

```diff
@@ -17,6 +17,8 @@ export const usePaywallLocks = (lockAddresses: string[]) => {
     })
 
     const locks = await Promise.all(lockPromises)
+    // getLock doesn't always return the lock address apparently
+    locks.forEach((lock, index) => (lock.address = lockAddresses[index]))
```

Comment:
Had to add this patch to `usePaywallLocks` because the locks I tried it on didn't include their addresses in the response.